include/swift/Serialization/Validation.h

```diff
@@ -91,6 +91,7 @@ struct ValidationInfo {
   version::Version compatibilityVersion = {};
   llvm::VersionTuple userModuleVersion;
   StringRef sdkName = {};
+  StringRef problematicRevision = {};
```

Comment:
All the `{}` aren't actually needed here - they'd already all be default initialized (like `VersionTuple`).